tests/unit/replybufsize.tcl

```diff
@@ -0,0 +1,56 @@
+proc get_reply_buffer_size {cname} {
+
+    set clients [split [string trim [r client list]] "\r\n"]
+    set c [lsearch -inline $clients *name=$cname*]
+    if {![regexp rbs=(\[a-zA-Z0-9-\]+) $c - rbufsize]} {
+        error "field rbus not found in $c"
+    }
+    return $rbufsize
+}
+
+start_server {tags {"replybufsize"}} {
```

Comment:
since all these tests are very slow, i'd rather add an `if {$::accurate)` which will prevent them from running in the normal execution of `./runtest`.

alternatively, maybe the 5 seconds interval we chose for the peak adjustment is too high? setting it to 1 second could have meant that this test will be over in 3 rather than 15.

Reply:
@yoav-steinberg I also agree. I suggested at some point that maybe we can allow tuning this feature in the DEBUG command?

Reply:
Yes, look at the implementation of `"key-load-delay"` config variable. This is probably where something like this should be (assuming @oranagra agrees).

Reply:
i'm not certain that this mechanism justifies a hidden config. (loading delay is heavily used by many tests).
I agree that generally, it would be wrong to change the threshold of the code just because we want the test to be fast, but in this case i'm not sure why we chose the 5 seconds threshold.
i remember arguing that we can't rely on the rate of the clientsCron since it could sometimes be much faster, but that doesn't mean we can't limit it to 1 second.
on the other hand, since this test it not that important, i don't mind running it only when `--accurate` is given (daily CI and before release)

Reply:
+1 for `--accurate`.